Directory.Build.targets

```diff
@@ -86,7 +81,7 @@
     </PropertyGroup>
   </Target>
 
-  <!-- This target will make sure that projects targeting netcoreapp3.1 will also have the Microsoft.Extensions.Logging.Abstractions analyzer removed. -->
+  <!-- This target will make sure that projects targeting net462 will also have the Microsoft.Extensions.Logging.Abstractions analyzer removed. -->
```

Comment:
@joperezr I kept this target because the comment in [Microsoft.Extensions.Telemetry.Abstractions.csproj](https://github.com/dotnet/extensions/pull/4394/files#diff-09674220f6f8ad9ca75c179401eeca831f746b20d7f1c11f099b46c7c72d7411) refers to both net462 and netcoreapp3.1:
![image](https://github.com/dotnet/extensions/assets/4403806/66e9d5ae-38d0-4b34-b927-e95d2a3cb40a)


Please let me know, if I misunderstood it, and these should be removed.